sonic-chassisd/scripts/chassisd

```diff
@@ -329,11 +376,106 @@ class ModuleUpdater(logger.Logger):
             midplane_ip = try_get(module.get_midplane_ip, default=INVALID_IP)
             midplane_access = try_get(module.is_midplane_reachable, default=False)
 
+            # Generate syslog for the loss of midplane connectivity when midplane connectivity
```

Comment:
A syslog when midplane connectivity is restored would be useful as well.

Reply:
Syslog added for midplane connectivity recovery.